### PR TITLE
log actual error when GetSrvKeyspace fails

### PR DESCRIPTION
**Pull request by deepthi**
Signed-off-by: deepthi <deepthi@planetscale.com>

### CODE REVIEW

File: go/vt/vttablet/tabletmanager/state_change.go [hunk 1]
```diff
@@ -220,7 +220,7 @@ func (agent *ActionAgent) changeCallback(ctx context.Context, oldTablet, newTabl
 			}
 			srvKeyspace, err := agent.TopoServer.GetSrvKeyspace(ctx, newTablet.Alias.Cell, newTablet.Keyspace)
 			if err != nil {
-				log.Error("Fail to get SrvKeyspace")
+				log.Errorf("Fail to get SrvKeyspace %v", err)
```
Comment (dweitzman) [inline go/vt/vttablet/tabletmanager/state_change.go]: Nit: seems like "Fail to get SrvKeyspace: %v" with ':' might be better than "Fail to get SrvKeyspace %v"

Comment (dweitzman) [reply go/vt/vttablet/tabletmanager/state_change.go]: Also: bonus points for changing this to "Failed" instead of "Fail"

**Comment by rafael** [comment]
Thank you for this fix! 

Per discussion in Slack, I think in this case we should squash these commits.

**Comment by deepthi** [comment]
> Thank you for this fix!
> 
> Per discussion in Slack, I think in this case we should squash these commits.

Done.

**Pull request merged**